Scripts/AssemblyInjections/Unity.Collections/FixedListInternalExtension.cs

```diff
@@ -12,7 +12,6 @@ namespace Anvil.Unity.Collections
     /// A collection of extension methods for <see cref="FixedList32Bytes{T}"/> (and friends) that require internal
     /// access to function.
     /// </summary>
-    [BurstCompatible]
     public static unsafe class FixedListInternalExtension
```

Comment:
Ok, I misunderstood how `[BurstCompile]` worked. That's fine.

I think comments might be helpful. It's fair enough that the IDE will warn you when calling non-burstable code from a bursted context but it doesn't really help when you're writing bursted code and poking around the codebase to see what you can find. Until you type it out (or read the whole class) you won't know if a piece of existing functionality that you've found will be compatible.

Maybe that doesn't happen often enough to justify writing comments and I'm inventing work here but I feel like, while writing bursted code, I'm often asking myself "can I use this method/type?"

Reply:
The counter-argument to comments is that it might be too general and the underlying code changes and whoever does that forgets to update the comment at the top of the file? 

In the scenario you're describing, I feel like the effort to type it out (which is pretty fast with autocomplete) to get the Burst warning in the IDE is the same or less then opening things up and reading comments which would be more to maintain? 

If we really wanted to signify to the developer that these are Burst safe things to use, I would argue that we should note that in the Class name or Method name instead?

Reply:
Let's try without the comments for now and see how it goes. 

When I'm devising a solution I'm not going to type out and check every method/type as I go. I'm going to collect a path or collection of types that "will work" and then go to implement.

Adding burst to all the type and method names is insane but if you're considering that I'll just drop this topic. You've backed me away from the ledge...